addons/mail/static/src/core/common/thread_model.js

```diff
@@ -819,7 +819,7 @@ export class Thread extends Record {
         } else {
             const tmpData = {
                 id: tmpId,
-                attachments: attachments,
+                attachment_ids: attachments,
```

Comment:
@tsm-odoo actually was in `tmpData` above but with typo, I made the minimal change here, not that important to make this change in 18.2 but if you think it's best let me know ;)